webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailsTable.js

```diff
@@ -77,6 +79,7 @@ const ContentViewVersionDetailsTable = ({
           autocompleteEndpoint,
           disableSearch,
         }}
+        ouiaId={`content_view_version_details_${snakeCase(route)}_table`}
```

Comment:
This produces a static id based on the route, like this:
<img width="1120" alt="Screen Shot 2022-03-21 at 10 03 11 AM" src="https://user-images.githubusercontent.com/38083295/160646831-ab488a17-aac5-43f6-bef5-3e02a60cdf51.png">

Reply:
After changes:
![Screen Shot 2022-03-29 at 9 50 41 AM](https://user-images.githubusercontent.com/38083295/160652926-0388a90d-552d-4920-a9bb-5bf70bcab1b0.png)